packages/core/src/components/Table/TableContainer/TableContainer.module.scss

```diff
@@ -7,7 +7,7 @@
     --cell-width: 40px;
     position: absolute;
     overflow: hidden;
-    left: calc(var(--cell-width) * -1);
+    left: calc(calc(var(--cell-width) * -1) + 1px);
```

Comment:
this has 0 effect on UI, and means that when leaving the row menu towards the row, there isn't a flicker, and no setTimeout needed to compensate on this few ms of moving from one item to another